prov/gni/src/gnix_msg.c

```diff
@@ -456,6 +439,29 @@ static int __recv_completion_src(
 	return FI_SUCCESS;
 }
 
+/*
+ * GNI provider generates a special CQE to indicate
+ * its releasing a FI_MULTI_RECV receive buffer back to
+ * the application.
+ */
```

Comment:
In the documentation, you write 'separate' which is indicated in the fi_cq documentation as allowed. In this segment, you write special. Which is it? Are we following spec and providing a separate CQE for notifying the application that the buffer is gone, or is this a GNI provider specific special CQE?